app/enterprise/changelog.md

```diff
@@ -4,6 +4,127 @@ no_search: true
 no_version: true
 skip_read_time: true
 ---
+## 2.3.0.0 (beta)
+**Release Date** 2020/01/20
+
+### **Kong Enterprise**
+The following sections list {{site.ee_product_name}}-exclusive updates,
+features, and fixes for the **2.3.0.0 beta** version.
+
+### Features
+
+#### Enterprise
+- Kong Enterprise can now run without a license in **Free Mode**. 
+  - In Free Mode, functionality includes **Kong Gateway (OSS)** and **Kong Manager**. 
+  - To enhance your Free Mode instance, you can add Enterprise functionality modules 
+  and features including **Kong Vitals**, **RBAC**, **Workspaces**, and **Kong Enterprise Plugins**. 
+  Contact your Kong sales representative for more information. 
+
+#### Core
+- Kong checks version compatibility between the control plane and any data planes to ensure 
+the data planes and any plugins have compatibility with the control plane in hybrid mode. 
+The sync is stopped if the major/minor version differ or if the installed plugin versions 
```

Comment:
Link to https://docs.konghq.com/2.3.x/hybrid-mode/#version-and-compatibility-checks

Reply:
> Link to https://docs.konghq.com/2.3.x/hybrid-mode/#version-and-compatibility-checks

I'd like to link to that, but 2.3.x on the OSS side is not present in @GayleNeumann 's branch since the PR was created before the 2.3 CE release. I'll try rebasing this branch. But this topic should be single-sourced @lena-larionova if it hasn't been already.

Reply:
Done in commit 9417c6a to OSS topic for now.